doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib

```diff
@@ -35,10 +35,64 @@ Sergio García Gómez (Telefónica I+D), Martin Bauer (NEC).
   
 ## Status
 
-This is a work in progress and is changing on a daily basis.
-Please send your comments to fiware-ngsi@lists.fiware.org.
-You can trace the discussions checking the archives of the mailing list:
-https://lists.fiware.org/private/fiware-ngsi/ (list subscription required).
+This specification is the final and stable version of NGSIv2 API specification (v2.0).
+
+# Changelog
+
+Changes since RC-2018.07:
+
+* None
+
+Changes since RC-2008.04:
```

Comment:
2008? :)

Reply:
:O

Reply:
Fixed in 45eb8eee53dd4193e688ec8d6e25b243611a7ba8